src/Commands/ListCommand.php

```diff
@@ -47,14 +47,16 @@ class ListCommand extends CustomerSecretsBaseCommand implements SiteAwareInterfa
      */
     public function listSecrets($site_id, array $options = ['debug' => false,])
     {
-        if ($this->getSite($site_id)) {
+        $this->setupRequest();
+        // @todo If terminus is pointing to sandbox then this should work.
+        //if ($this->getSite($site_id)) {
```

Comment:
Doesn't the `listSecrets` API call check to see if the active session has access to the specified site ID?  i.e. you should get an error if you use an invalid site ID. I don't think it's necessary to have the plugin call `getSite` to check.

Reply:
makes sense 👍